### DISCUSSION TITLE
connected players on the Overview page

### DISCUSSION THREAD

**Discussion by ssiwm** [gameserverapp/Platform#480] [Suggestions]
I think it would be a good idea to add a list of connected players on the Overview page

**Comment by mvaessen** [comment]
Hi, you should be able to add this already, or I misunderstand your suggestion

**Reply by LimitlessDrag0n** [reply]
I guess in the server overview page not the dashboard?

**Comment by ssiwm** [comment]
that's right, it would be nice to see players connected to the server on the server Overview page.
